.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+test.md
```

Comment:
- [ ] Please add  the test.md correctly to `gitignore`. I do see it here but also is in you repo.

Reply:
how to do it  correctly so please help me